test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/CsdlWriterTests.cs

```diff
@@ -1986,7 +1986,7 @@ public void CanWriteNavigationPropertyBindingWithTargetPathOnContainmentOnSingle
                     "<EntityContainer Name=\"Default\">" +
                        "<Singleton Name=\"Me\" Type=\"NS.Customer\" />" +
                        "<EntitySet Name=\"Customers\" EntityType=\"NS.Customer\">" +
-                         "<NavigationPropertyBinding Path=\"ContainedOrders/OrderLines\" Target=\"Me/ContainedOrderLines\" />" +
+                         "<NavigationPropertyBinding Path=\"ContainedOrders/OrderLines\" Target=\"NS.Default/Me/ContainedOrderLines\" />" +

```

Comment:
We should not be container-qualifying the target unless necessary (which it shouldn't be)

Reply:
Revert to path without qualified name

---
In reply to: [542983048](https://github.com/OData/odata.net/pull/1950#discussion_r542983048) [](ancestors = 542983048)